packages/nextjs/src/client/browserTracingIntegration.ts

```diff
@@ -12,6 +12,16 @@ export function browserTracingIntegration(
     ...options,
     instrumentNavigation: false,
     instrumentPageLoad: false,
+    onRequestSpanStart(...args) {
```

Comment:
l: do we care about cases where users register their own `browserTracingIntegration()`? As in should we merge a possible custom implementation with our logic? Not sure what our general approach to this is so feel free to declare undefined behaviour :D

Reply:
I think user's would be always importing this `browserTracingIntegration` (the Next.js one) so the behavior should already be merged (see line 23). Unless I am misunderstanding.